components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtDBQueries.java

```diff
@@ -27,9 +27,9 @@
 
     // STORE Queries
     public static final String STORE_BASIC_APPINFO = "INSERT INTO SP_APP (TENANT_ID, APP_NAME, USER_STORE, USERNAME, " +
-            "DESCRIPTION, AUTH_TYPE) VALUES (?,?,?,?,?,?)";
+            "DESCRIPTION, AUTH_TYPE, IS_DUMB_MODE) VALUES (?,?,?,?,?,?,?)";
```

Comment:
Cannot/no need to be part of this query. We always store the basic information of the application first and then update the information with other details. That is why we don't have any other variables in this query.